.gitignore

```diff
@@ -1 +1,3 @@
 .DS_Store
+
+coinlib/src/
```

Comment:
This shouldn't be necessary any more.

Reply:
Resolved by c416cd3efa87a3f0d396ef3ae5df46559088e385